test/uk/gov/hmrc/gform/sharedmodel/FormComponentRejectSpec.scala

```diff
@@ -103,11 +103,8 @@ class FormComponentRejectSpec extends Spec with TableDrivenPropertyChecks {
       ("json",                                     "expectedMessage"),
       ("invalid-count-usage",                      "sections.fields.[id=lastInfo].infoText: textA cannot be use with .count function. Only AddToList id can be used with .count"),
       ("invalid-sum-usage",                        "sections.fields.[id=lastInfo].infoText: textA cannot be use with .sum function. Only numeric fields from Group component, Repeated section or AddToList section can be used with .sum function"),
-      ("invalid-addtolist-field-reference",        "title: fieldA belongs to Add To List section and cannot be referenced outside"),
       ("invalid-addtolist-cross-reference",        "title: fieldA belongs to different Add To List section"),
-      ("invalid-repeated-section-field-reference", "includeIf: fieldA belongs to Repeated section and cannot be referenced outside"),
       ("invalid-repeated-section-cross-reference", "fields.[id=fieldB].label: fieldA belongs to different Repeated section"),
-      ("invalid-group-field-reference",            "includeIf: fieldA belongs to Group component and cannot be referenced outside"),
```

Comment:
Please delete corresponding json files if they are not invalid anymore.

Reply:
done